### ISSUE TITLE
Request new feature: Allow to select functions to stream es logs 

### ISSUE THREAD

**Issue by snapify-tech** [daniel-cottone/serverless-es-logs#321]
Current status: 
for the serverless project include more than 2 functions, after deploying, all functions are set up subscriptions. But some of them are not required to stream CloudWatch logs to ES.

Request Feature:
allow selecting a list of functions to apply the "serverless-es-logs"

Thanks

**Comment by owain68** [comment]
I agree that a filter on function name would be a great help.  Not all functions necessarily need to stream to ES.
